examples/media/schema.json

```diff
@@ -0,0 +1,299 @@
+{
+  "name": "rsi.service.media",
+  "description" : "",
+  "$schema": "https://github.com/github/wzr1337/blob/2.0.0/dist/$rsi.schema.json",
+  "resources" : [
+    {
+      "name": "collections",
+      "description": "The media collections can also be understood as media queues of the renderer. Media collections can only contain playable media like tracks, videos or pictures.",
+      "objectModel": {
+        "id": {
+          "description": "collection id",
+          "type": "string",
+          "format": "uuid"
+        },
+        "name": {
+          "description": "collection name",
+          "type": "string"
+        },
+        "uri": {
+          "description": "object uri",
+          "type": "string",
+          "format": "uri"
+        },
+        "items": {
+          "description": "media collection items",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "oneOf": [
+            {
+              "#ref": "medialibrary.videosObject"
+            },
+            {
+              "#ref": "medialibrary.picturesObject"
+            },
+            {
+              "#ref": "medialibrary.tracksObject"
+            }
+            ]
+          }
+        }
+      },
+      "endpoints": {
```

Comment:
I don't like the endpoints property. In the protocol, we have defined that everything such as service, resource and element is accessible from the URL by its name or id. But the schema doesn't reflect the protocol definition.

Reply:
https://github.com/wzr1337/rsi.schema/issues/6